p2p/p2p.proto

```diff
@@ -38,8 +38,17 @@ message AnchorDocumentResponse {
   bool accepted = 1;
 }
 
+enum AccessType {
+  ACCESS_TYPE_INVALID = 0;
+  ACCESS_TYPE_REQUESTER_VERIFICATION = 1;
+  ACCESS_TYPE_NT_OWNER_VERIFICATION = 2;
+  ACCESS_TYPE_ACCESS_TOKEN_VERIFICATION = 3;
+}
+
 message GetDocumentRequest {
   bytes document_identifier = 1;
+  AccessType access_type = 2;
+  bytes nft = 3;
```

Comment:
Add a comment that NFT bytes = 20 bytes of NFT registry + 32 bytes of TokenID

or have two different fields for each

Reply:
I'll put this into two separate fields, since this is how the NFTOwnerCanRead method evaluates them for Read ACLs.

Reply:
Awesome 👍